### DISCUSSION TITLE
Git Help

### DISCUSSION THREAD

**Discussion by ivavay** [mdwiltfong/secret-santa#57] [General]
Hi @mdwiltfong! I'm trying to make sure the files on my computer are up to date with the changes that's been made in the original repository on your end. I think I'm supposed to 'sync fork' but it says I need to either discard my 7 commits or get my PR approved (which I think it's still in draft right now). That draft one was mostly some navbar fixes and giftshop page.

**Comment by mdwiltfong** [comment]
Hey @ivavay ! 

@freddieb77 had a similar issue when he made commits directly to the `main` branch of fork. Looking at the 7 commits on your `main` branch, could you discard them? Does GitHub give you that option or is it git on your machine that is telling you to discard the 7 commits?

**Reply by ivavay** [reply]
Ahh yeah I can do that! Yeah Github does give me that option.
